### PR TITLE
Firefox 77 ships String.prototype.replaceAll()

### PR DESCRIPTION
**Pull request by bershanskiy**
Sources:
 - https://bugzil.la/1608168
 - SpiderMonkey Newsletter 4
   https://mozilla-spidermonkey.github.io/blog/2020/05/11/newsletter-4.html

### CODE REVIEW

File: javascript/builtins/String.json [hunk 1]
```diff
@@ -1961,6 +1961,58 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": true,
```
Comment (Elchi3) [inline javascript/builtins/String.json:2010]:
```suggestion
              "experimental": false,
```

File: javascript/builtins/String.json [hunk 2]
```diff
@@ -1961,6 +1961,58 @@
...
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
```
Comment (Elchi3) [inline javascript/builtins/String.json:1997]:
```suggestion
                "version_added": "13.1"
```

File: javascript/builtins/String.json [hunk 3]
```diff
@@ -1961,6 +1961,58 @@
...
+              },
+              "safari_ios": {
+                "version_added": false
```
Comment (Elchi3) [inline javascript/builtins/String.json:2000]:
```suggestion
                "version_added": "13.4"
```

**Comment by bershanskiy** [comment]
Thanks for review! Does it look better now?

**Pull request merged**